### ISSUE TITLE
RC Test: Build An Atom 1.4.0-phetiorc.2

### ISSUE THREAD

**Issue by jbphet** [phetsims/tasks#850]
A new RC has been released for this that removes the game screen, see https://github.com/phetsims/build-an-atom/issues/159.  

This issue supersedes https://github.com/phetsims/tasks/issues/847.

Here is the link to the RC version that should be tested:
http://www.colorado.edu/physics/phet/dev/html/build-an-atom/1.4.0-phetiorc.2/wrappers/index/

The scope of the testing has been discussed in recent meetings and I think @phet-steele is pretty aware of what needs to be done.

**Comment by phet-steele** [comment]
Testing is done, back to @zepumph. Features in #847 were verified as (mostly) working.

**Comment by zepumph** [comment]
#850 should be addressed here too.

**Comment by zepumph** [comment]
closing, next rc: https://github.com/phetsims/tasks/issues/859
